src/BTCPayServer.Lightning.CLightning/ShortChannelId.cs

```diff
@@ -0,0 +1,29 @@
+using System.Globalization;
+using System.Linq;
+
+namespace BTCPayServer.Lightning.CLightning
+{
+    // clightning-specific representation of channel id
+    public class ShortChannelId
```

Comment:
can you override equal, gethashcode and == operators? Like I do normally for `LightMoney`.